<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-U84CKO6" ignored="0" name="Test1" tests="2" failures="0" timestamp="2020-02-04T12:54:18 IST" time="0.062" errors="0">
  <testcase name="@BeforeTest configBT" time="0.015" classname="Grid.Demo">
    <failure type="java.net.MalformedURLException" message="no protocol: ">
      <![CDATA[java.net.MalformedURLException: no protocol: 
at java.net.URL.<init>(Unknown Source)
at java.net.URL.<init>(Unknown Source)
at java.net.URL.<init>(Unknown Source)
at Grid.Demo.configBT(Demo.java:21)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeTest configBT -->
  <testcase name="sample" time="0.0" classname="Grid.Demo">
    <skipped/>
  </testcase> <!-- sample -->
  <testcase name="sample" time="0.0" classname="Grid.Demo">
    <skipped/>
  </testcase> <!-- sample -->
</testsuite> <!-- Test1 -->
